requirements/edx/github.in

```diff
@@ -101,4 +101,5 @@
 
 -e git+https://github.com/mitodl/edx-sga.git@3828ba9e413080a81b907a3381e5ffa05e063f81#egg=edx-sga==0.8.3
 -e git+https://github.com/open-craft/xblock-poll@add89e14558c30f3c8dc7431e5cd6536fff6d941#egg=xblock-poll==1.5.1
+-e git+https://github.com/open-craft/xblock-html.git@jazzar/initial-changes#egg=xblock-html==1.0.0
```

Comment:
Was this deliberately added?  I don't see it reflected in the `*.txt` files, and I thought the main point of the PR was to upgrade bleach so this could be added later.

Reply:
Good catch, `xblock-html` should not be included for now.

Reply:
Good catch! Committed by mistake when I was testing the upgraded bleacher with external XBlocks.